did/resolver/resolver.go

```diff
@@ -1,12 +1,15 @@
-package did
+package resolver
```

Comment:
To avoid stuttering, and seems like a better name for the package. 
```suggestion
package resolution
```

Reply:
good idea